src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionServiceTest.java

```diff
@@ -602,4 +605,21 @@ void deleteAttachmentHandleNullHttpStatusApiResponseOnDeleteAttachment() {
                 any());
     }
 
+    @Test
+    public void willCallFileTransferGatewayForDownload() throws IOException, ServiceException {
+        HttpServletResponse httpServletResponse = new MockHttpServletResponse();
+        FileTransferApiClientResponse dummyDownloadResponse = Utils.dummyDownloadResponse();
+
+        when(fileTransferApiClient.download(ATTACHMENT_ID, httpServletResponse)).thenReturn(dummyDownloadResponse);
+
+        FileTransferApiClientResponse downloadServiceResult = objectionService.downloadAttachment(
+                REQUEST_ID, OBJECTION_ID, ATTACHMENT_ID, httpServletResponse);
+
+        verify(fileTransferApiClient, only()).download(ATTACHMENT_ID, httpServletResponse);
+        verify(fileTransferApiClient, times(1)).download(ATTACHMENT_ID, httpServletResponse);
+
+        assertNotNull(downloadServiceResult);
```

Comment:
Can we check that the status of the result is the same as the dummyDownloadResponse? Just to confirm the ObjectionService doesn't modify it

Reply:
Now checking the status